src/vs/workbench/common/editor/editorGroupModel.ts

```diff
@@ -403,7 +415,7 @@ export class EditorGroupModel extends Disposable implements IEditorGroupModel {
 
 			// Handle active
 			if (makeActive) {
-				this.doSetActive(newEditor, targetIndex);
+				this.doSetActive(newEditor, targetIndex, options?.selected);
```

Comment:
What if I want to open an editor inactive but selected? Don't we have to apply `options.selected` always?

Reply:
Currently not possible to happen but for the model to be complete, I'll add support for it.